add transport mode and vehicle, models and views

lepistone opened the pull request:
- [x] models
- [x] views
- [x] doc
- [x] i18n

lepistone committed:
add transport mode and vehicle, models and views

lepistone committed:
make mode and vehicle translatable, add POT

lepistone committed:
add doc to transport_information

yvaucher commented:
:+1:

gurneyalex commented on transport_information/README.rst:
```diff
@@ -0,0 +1,10 @@
+Transport Information
+=====================
+This simple module is en extraction of objects related to transportation of
```
s/en/an/

gurneyalex commented on transport_information/model/__init__.py:
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+import transport_mode
+import transport_vehicle
```
please use relative imports

gurneyalex commented on transport_information/security/ir.model.access.csv:
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_transport_mode,access_transport_mode,model_transport_mode,,1,1,1,1
+access_transport_vehicle,access_transport_vehicle,model_transport_vehicle,,1,1,1,1
```
I'm concerned that such permissions could allow e.g. portal users to create transport modes and vehicles. IMO, by default only purchase managers should be allowed to create such records (and anyone can read them)

foutoucour replied:
I agree here.
I would add to the suggestions from @gurneyalex that only admin should be able to unlink.

lepistone committed:
fix typo in README

lepistone committed:
use relative imports

lepistone committed:
tighten permissions of transport mode/vehicle

It is not sensible information, so I am giving anyone the right to read
and purchase managers the right to do anything.

lepistone commented:
I addressed the points raised. I gave purchase managers (not admin) the right do delete those.

Thanks for the review.
